server/.env.example

```diff
@@ -1,7 +1,7 @@
 # General site settings
 APP_ENV=local
 APP_DEBUG=true
-APP_KEY=SomeRandomString
+APP_KEY=SomeRandomStringSomeRandomString
```

Comment:
It does need to be longer for the new default encryption setting to work. I'll update to include a clarification.